### PR TITLE
Add CircleCI support

### PR DESCRIPTION
**Pull request by radekk**
Replace TravisCI with CircleCI.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
```
Comment (esarafianou) [inline .circleci/config.yml]: Do we support only node 10?

Comment (radekk) [reply .circleci/config.yml]: We were running on v6 and then migrated over to v8 when webtask upgrade node.js. I believe that repo supervisor should be compatible with Node.js v6+ all the way to the latest versions.

Comment (esarafianou) [reply .circleci/config.yml]: Shouldn't we run the tests for all the major node versions we support?

Comment (radekk) [reply .circleci/config.yml]: We might think about it in the future since it's not set in stone which versions we want to support yet. 👍

Comment (esarafianou) [reply .circleci/config.yml]: Sounds good.

**Pull request merged**